SimplyPluralApiBodies.yml

```diff
@@ -526,6 +500,54 @@ components:
             type: string
             description: Id of the member
 
+    groupContainer:
+      type: object
+      properties:
+        exists:
+          type: boolean
+        id:
+          description: The group Id
+          type: string
+        content:
+          description: The group content
+          $ref: "#/components/schemas/groupInfo"
+
+    groupInfo:
+      type: object
+      properties:
+        $ref: "#/components/schemas/group/properties"
+        buckets:
+          description: The privacy buckets for this group
+          type: array
+          items:
+            type: string
+        avatarUuid:
+          description: Id of the uploaded avatar
+          type: string
+        frame:
+          $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/frame"
+        uid:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/description"
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/uid/type"
+        lastOperationTime:
+          description:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/description"
+          type:
+            $ref: "./SimplyPluralApiCommons.yml#/components/schemas/commons/properties/lastOperationTime/type"
+
+    groupSend:
+      type: object
+      properties:
+        $ref: "#/components/schemas/group/properties"
+        private:
```

Comment:
Should probably mention on "private" and "preventTrusted" properties that they're all legacy, so people don't get confused when they don't do anything on new/updated accounts